write_spectra/write_spectra.py

```diff
@@ -0,0 +1,268 @@
+
+import os
+import sys
+import logging
+from pandas import to_datetime
+from astropy.time import Time
+from urllib.parse import unquote
+from datetime import date
+import astropy.io.fits as fits
+from astropy.table import Table
+import numpy as np
+from matplotlib import pyplot as plt
+from specutils import Spectrum1D
+
+#first generate wavelength, flux, and error arrays
+#NIRspec_wavelength = []
+#NIRspec_flux = []
+#NIRspec_error = []
+
+#MIRI_wavelength = []
+#MIRI_flux = []
+#MIRI_error = []
+
+#Make a JWST Header w smae info, and then 2 w MIRI AND NIRSpec specific info
+#1-5 microns (NIRSpec) and 5-20 microns (MIRI)
+
+
+#'fits_data_dir': '/Users/fits_output_path',  # set outside the dictionary
+#'generated_filename': ' '
+
+"""JWST_header_dict = {
+    # Information about new spectra
+    'RA': '',
+    'Dec': '',
+    'generated_history': 'Data Reduced by JWST pipeline version 1.7.2, CRDS version 11.16.12, CRDS context jwst_0977.pmap', #get
+    'object_name' : 'VHS1256b ',
+    'telescope': 'JWST',
+     }"""
+
+#NIRSpec_generated_filename: ' '
+"""NIRSpec_header_dict = {
+    'instrument': 'NIRSpec',
```

Comment:
Let's also add the bandpass. I don't remember if we want to use the UCD names or not, but for now, lets say bandpass = "NIR" for NIRSPEC and "MIR" for MIRI.